### ISSUE TITLE
Page isn't read properly using VO in iOS

### ISSUE THREAD

**Issue by paulettefrancis** [cds-snc/covid-shield-mobile#292]
When reading using VoiceOver, "COVID Alert" (Tap for menu) from the "No exposure detected" page, it doesn't read the "menu" page, but reads what's underneath.

iPhone XR 13.5.1

![image](https://user-images.githubusercontent.com/22638158/86050898-84ecfe80-ba22-11ea-9af6-48bcd5a7ff05.png)

**Comment by JuliannaR** [comment]
Closed because this is duplicated in Collapse BottomSheet after submission flow is finished #387
